client/sites-dashboard/components/wpcom-smp-commands.tsx

```diff
@@ -577,6 +584,7 @@ export const useCommandsArrayWpcom = ( {
 					navigate( `/hosting-config/${ site.slug }` );
 				},
 				filter: ( site: SiteExcerptData ) => site?.is_wpcom_atomic,
+				filterNotice: __( 'Only listing sites with hosting features enabled.' ),
```

Comment:
The filter is hiding the P2s and Jetpack sites.
Simple sites has the upsell flow.

```suggestion
				filterNotice: __( 'Only listing sites hosted on WordPress.com.' ),
```